styles.js

```diff
@@ -90,3 +92,38 @@ export const NoData = styled.div`
   align-items: center;
   box-shadow: 1px 1px 2px;
 `;
+
+export const SelectionContainer = styled.section`
+  background-color: rgb(255, 255, 255, 0.1);
+  border-radius: 10px;
+  padding: 1rem;
+`;
+
+export const GamesContainer = styled.section`
+  border-radius: 10px;
+  padding: 1.5rem;
+  margin: 0.2rem;
+`;
+
+export const StyledSelect = styled(Select)`
+  width: 100px;
+  height: 40px;
+  font-size: 16px;
+  align-self: center;
+`;
+
+export const GamesList = styled.ul`
+  list-style: none;
+  background-color: rgb(255, 255, 255, 0.2);
+  border-radius: 10px;
+  padding: 0.5rem;
+  display: grid;
+  justify-content: center;
+`;
+
+export const SingleGame = styled.li`
+  font-size: 0.8rem;
+  margin-bottom: 0.6rem;
+  margin-top: 0.6rem;
+  font-size: 1rem;
```

Comment:
Only one `font-size` is needed I suppose ;)
```suggestion
  margin-bottom: 0.6rem;
  margin-top: 0.6rem;
  font-size: 1rem;
```